cosmos/azure-spring-data-cosmos/spring3-samples/cosmos-aad-sample/README.md

```diff
@@ -0,0 +1,146 @@
+---
+page_type: sample
+languages:
+- java
+products:
+- azure-cosmos-db
+name: Using Azure Cosmos DB RBAC by Spring Data
+description: This sample demonstrate basic Spring Data code for Java SQL API to connect to Azure Cosmos DB using built-in role-based access control (RBAC), and authenticating using Azure AD.
+---
+
+# Using Azure Cosmos DB RBAC by Spring Data
+
+[Azure Cosmos DB](https://learn.microsoft.com/azure/cosmos-db/introduction) [Spring Data](https://spring.io/projects/spring-data) [RBAC](https://en.wikipedia.org/wiki/Role-based_access_control) with [Azure AD](https://learn.microsoft.com/azure/active-directory/fundamentals/active-directory-whatis) Sample project.
+
+## Features
+
+This sample demonstrate basic Spring Data code for Java SQL API to connect to Azure Cosmos DB using built-in role-based access control (RBAC), and authenticating using Azure AD. See instructions below on setting up the RBAC/Azure AD requirements to access your Cosmos DB account and run the app successfully.
```

Comment:
Should we add some descriptions to tell people this sample only work with sb3?
Same for other samples, same for sb2.

Reply:
Added to all readme's for SB2 and SB3.